src/main/java/io/fusionauth/http/server/HTTPServerThread.java

```diff
@@ -278,9 +278,14 @@ private void cleanup() {
         continue;
       }
 
-
       if (logger.isDebugEnabled()) {
-        logger.debug("Closing connection readingSlow=[{}] writingSlow=[{}] timedOut=[{}]", readingSlow, writingSlow, timedOut);
+        String message = readingSlow
```

Comment:
Chained ternaries are easy to mess up for the future. I think it's worth a bit more verbosity using `if/else if` to prevent future headaches.

```
String message = "";
if (readingSlow) {
  message = ...;
} else if (writingSlow) {
  message = ...;
}
```